app/controllers/front_ui_controller.rb

```diff
@@ -44,6 +44,20 @@ def save_location
 
   def about; end
 
+  def location
+    @loc = params[:loc].downcase
+
+    @maps = Map.page(params[:maps])
+               .per_page(20)
+               .where('archived = ?, password = ?, location LIKE ?', false, '', "%#{@loc}%")
```

Comment:
```suggestion
               .where('archived = ? and password = ? and location LIKE ?', false, '', "%#{@loc}%")
```

it will solve the 500 error whose screenshot I pasted

Reply:
Made the changes. Actually I was not able to run it locally after Rail 5 merge. It was showing me the error: ```NameError: uninitialized constant Recaptcha::ClientHelper``` 
So I couldn't test it.

Reply:
Oh, that's the same issue as in https://github.com/publiclab/mapknitter/issues/953 ??? Let's ping folks there!